samples/mimxrt1060/basic-sample/include/app_config.h

```diff
@@ -2,22 +2,15 @@
 // Copyright: Avnet 2021
 // Created by Nik Markovic <nikola.markovic@avnet.com> on 4/19/21.
 //
-
 #ifndef APP_CONFIG_H
 #define APP_CONFIG_H
-
-#define WIFI_SSID       "your wifi ssid"
-#define WIFI_PASSWORD   "your wifi password"
-
-#define IOTCONNECT_CPID "your cpid"
-#define IOTCONNECT_ENV  "your environment"
-
-//#define IOTCONNECT_DUID "custom_DUID" // you can supply a custom device UID, or...
+#define WIFI_SSID       ""
+#define WIFI_PASSWORD   ""
+#define IOTCONNECT_CPID "avtds"
+#define IOTCONNECT_ENV  "Avnet"
+#define IOTCONNECT_DUID "fae-rt1060" // you can supply a custom device UID, or...
 #define DUID_PREFIX "nxp-" // mac address will be appended in format 012345abcdef
-
 // if it is not defined, device certs will be used in iotconnect_app.c
-#define IOTCONNECT_SYMETRIC_KEY "symetric key"
-
+#define IOTCONNECT_SYMETRIC_KEY "c2h1aXNteWJlc3RmcmllbmQ="
```

Comment:
Key checked in - probably not intended.

Reply:
Fixed it.

Reply:
@faceface1234 DUID cannot have spaces. I fixed it.